frontend/src/layouts/DefaultLayout.vue

```diff
@@ -1,30 +1,152 @@
 <template>
   <v-main class="bg-background main-layout">
-    <TopMenu />
-    <v-container fluid class="page-container px-8">
-      <v-row>
-        <v-col v-if="slots.sidemenu" cols="2">
-          <v-sheet rounded>
-            <slot name="sidemenu"></slot>
-          </v-sheet>
+    <!-- Top Menu -->
+    <v-app-bar flat class="app-bar" height="70">
+      <v-row class="ma-1">
+        <v-col class="d-none d-md-flex align-center">
+          <a href="/" class="logo">
+            <LogoImage size="small" />
+          </a>
         </v-col>
-
-        <v-col class="pa-0 pa-sm-1 pt-0">
-          <slot />
+        <v-col class="d-flex align-center justify-center">
+          <TabControl />
+        </v-col>
+        <v-col class="d-none d-md-flex align-center">
+          <v-row>
+            <v-col class="d-flex align-center">
+              <LightDarkSwitch class="d-none d-lg-flex" />
+            </v-col>
+            <v-col class="d-flex align-center justify-end">
+              <button @click="toggleDrawer">
+                <Circle>
+                  <v-icon icon="$camera"></v-icon>
+                </Circle>
+              </button>
+              <UserInfo class="ml-2" />
+            </v-col>
+          </v-row>
         </v-col>
       </v-row>
+    </v-app-bar>
+    <TablesDrawer
+      v-model="isTablesDrawerVisible"
+      :location="$vuetify.display.smAndDown ? 'bottom' : 'right'"
+    />
+
+    <!-- Page Content Container -->
+    <v-container fluid class="page-container px-8">
```

Comment:
```suggestion
    <div class="page-container px-8">
```
VContainer is a part of Vuetify's grid system, which we are not using here.
https://vuetifyjs.com/en/components/grids/